src/main/java/com/plotsquared/plothider/storage/BlockStorage.java

```diff
@@ -0,0 +1,228 @@
+/*
+ *              _____  _       _   _    _ _     _
+ *              |  __ \| |     | | | |  | (_)   | |
+ *              | |__) | | ___ | |_| |__| |_  __| | ___ _ __
+ *              |  ___/| |/ _ \| __|  __  | |/ _` |/ _ \ '__|
+ *              | |    | | (_) | |_| |  | | | (_| |  __/ |
+ *              |_|    |_|\___/ \__|_|  |_|_|\__,_|\___|_|
+ *               PlotHider PlotSquared addon for Minecraft
+ *              Copyright (C) 2016 - 2022 IntellectualSites
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.plotsquared.plothider.storage;
+
+import com.plotsquared.plothider.storage.palette.APalettedContainer;
+import com.plotsquared.plothider.storage.palette.PalettedContainerType;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class BlockStorage {
+    private short blockCount;
+    private FlexibleStorage storage;
+
+    public BlockStorage(short blockCount) {
+        this.blockCount = blockCount;
+        this.storage = new FlexibleStorage();
+    }
+
+    public FlexibleStorage getStorage() {
```

Comment:
Unused?

Reply:
All these getters/setters have been generated automatically. Should I removed them if unused?

Reply:
I see no need to add something that is not used at all.